python/tvm/relax/frontend/torch/fx_translator.py

```diff
@@ -535,7 +551,13 @@ def _cat(self, node: fx.node.Node) -> relax.Var:
 
     def _expand(self, node: fx.node.Node) -> relax.Var:
         args = self.retrieve_args(node)
-        return self.block_builder.emit(relax.op.broadcast_to(args[0], args[1:]))
+        broadcast_shape, in_shape = [], self.shape_of(args[0])
```

Comment:
Please add a testcase for when -1 is given as expand dim.
https://github.com/apache/tvm/blob/b76ebad8867e36121708cf654923b66c4f7c9ede/tests/python/relax/test_frontend_from_fx.py#L2970

Reply:
done